draft-ietf-rtcweb-security-arch.xml

```diff
@@ -556,6 +556,163 @@
+         an opaque string that is consumed by the IdP. The <spanx style="verb">"idp"</spanx> key value contains a
+         dictionary with one or two further values that identify the IdP. See
+         <xref target="sec.request-assert"/> for more details.
+       </t>
+       <section title="Offer/Answer Considerations" anchor="sec.sdp-id-attr-oa">
+         <t>
+		   This section defines the SDP Offer/Answer <xref target="RFC6454"/> considerations for the SDP
+		   'identity' attribute.
+	     </t>
+         <t>
+           Within this section, 'initial offer' refers to the first offer, within an
+           SDP session, in which the offerer includes an SDP 'identity attribute'.
+         </t>
+         <section title="Generating the Initial SDP Offer" anchor="sec.sdp-id-attr-oa-inio">
+         <t>
+           When an offerer sends an offer, in order to provide its identity
+           assertion to the peer, it includes an 'identity' attribute in the
+           offer. In addition, it MUST include SDP 'fingerprint' attributes for
+           each fingerprint that has been used to assert the identity.
+         </t>
+         </section>
+         <section title="Answerer Processing of SDP Offer" anchor="sec.sdp-id-attr-oa-anso">
+         <t>
+           When an answerer receives an offer that contains an 'identity'
+           attribute, the answerer can use the the attribute information to
+           contact the IdP, and verify the identity of the peer.  If the
+           answerer does not accept the attribute value, or if the identity
+           verification fails, it MUST reject the offer.
+         </t>
+         </section>
+         <section title="Generating of SDP Answer" anchor="sec.sdp-id-attr-oa-ansa">
+         <t>
+           When an answerer sends an answer, in order to provide its identity
+           assertion to the peer, it includes an 'identity' attribute in the
+           answer. In addition, it MUST include 'fingerprint' attributes for
+           each fingerprint that have been used to assert the identity.
+         </t>
+         <t>
+           NOTE: The answerer can include an 'identity' attribute in the answer
+           independently of whether the corresponding offer contained an
+           identity attribute or not.
+         </t>
+         </section>
+         <section title="Offerer Processing of SDP Answer" anchor="sec.sdp-id-attr-oa-offa">
+         <t>           
+           When the offerer receives an answer that contains an 'identity'
+           attribute, the offerer can use the attribute information to contact
+           the IdP, and verify the identity of the peer. If the offerer does
+           not accept the attribute (or the attribute value), or if the identity
+           verification fails, it MUST take proper actions to reject the answer.
+         </t>
+         </section>
+         <section title="Modifying Session" anchor="sec.sdp-id-attr-oa-modi">
```

Comment:
"Modifying an Existing Session" perhaps

Reply:
We have previously used "Modifying The Session", so I suggest to use that.